app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> any:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> any:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income_for_wash = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income_for_wash += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return income_for_wash
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round((car.comfort_class
+                     * (self.clean_power
+                        - car.clean_mark)
```

Comment:
Can you split such huge formula into smaller chunks to increase readability of your code?